ejabberd_bridge/management/commands/ejabberd_auth.py

```diff
@@ -58,6 +59,8 @@ def to_ejabberd(self, answer=False):
     def auth(self, user_id=None, server="localhost", token=None):
         self.logger.debug("Authenticating %s on server %s" % (user_id, server))
         try:
+            user_id = int(user_id)
```

Comment:
@fichie23 this should catch ValueError, and return False instead of forward the exception